staff/asier/react-spotify-api-promises/scripts/spotify-api-1.0.0.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Spotify API client.
+ *
+ * @version 1.0.0
+ */
+
+let spotifyApi;
+
+(function () {
+    "use strict";
+    const baseUrl = "https://api.spotify.com/v1/"
+    const token = 'BQD4yuFe_0go2HTaHXYd-kPN1LMOFnKGVNYq0HufjYlp5byMKWCVIe2GLyS8DamSAEfwrAuZhhlY2JymuiMu3kAwrHVfe_uht2W5c_89fNGJMVGI9Kto2XVUg1LbP5-6s9bkaJ0'
+    const headers = { Authorization: 'Bearer ' + token }
+
+    
+    spotifyApi = {
+        baseUrl,

```

Comment:
SEE notes already provided in this same file located in design. they also apply here ,)